### PR TITLE
Fix language server complaints

### PR DESCRIPTION
**Pull request by danielbernard**
pedantry

### CODE REVIEW

File: modules/towns/registration_npc.dsc [hunk 1]
```diff
@@ -52,15 +52,17 @@ cc_towns_registrar_setup_t:
...
     type: command
+    description: Initializes the settlements flag in Denizen.
     name: setupsettlements
     usage: /setupsettlements
     permission: group.admin
     script:
-        - if <server.has_flag[settlements]> == false:
-            - flag <server> settlements:<map.with[index].as[0]>
+        - if !<server.has_flag[settlements]>:
+            - flag server settlements:<map.with[index].as[0]>
```
Comment (danielbernard) [inline modules/towns/registration_npc.dsc]: Accidentally introduced a change here; I'll revert it in a moment

File: modules/towns/registration_npc.dsc
Comment (mmsheeks) [inline modules/towns/registration_npc.dsc:1]: We can actually just remove this file. We aren't using this NPC.

Comment (mmsheeks) [reply modules/towns/registration_npc.dsc:1]: Or rather, we can remove the `cc_towns_setup_c` script, specifically.

Comment (mmsheeks) [reply modules/towns/registration_npc.dsc:1]: This isn't important enough to hold up this merge, thanks for the cleanup bud.

**Pull request merged**